src/instructlab/model/convert.py

```diff
@@ -1,127 +1,115 @@
 # SPDX-License-Identifier: Apache-2.0
 
+# Future
+from __future__ import annotations
+
 # Standard
-from glob import glob
+from enum import Enum
 from pathlib import Path
+from typing import Callable, TypeVar
 import logging
-import os
-import shutil
-
-# Third Party
-from huggingface_hub import errors as hf_errors
-from requests import exceptions as requests_exceptions
-import click
 
 # First Party
-from instructlab import clickext, utils
+from instructlab.utils import is_model_gguf, is_model_safetensors
 
 logger = logging.getLogger(__name__)
 
 
-@click.command()
-@click.option(
-    "--model-dir",
-    help="Base directory where model is stored.",
-    show_default=True,
-    required=True,
-)
-@click.option("--adapter-file", help="LoRA adapter to fuse.", default=None)
-@click.option(
-    "-sd",
-    "--skip-de-quantize",
-    help="Skip de-quantization.",
-    is_flag=True,
-)
-@click.option(
-    "-sq",
-    "--skip-quantize",
-    is_flag=True,
-    help="Whether to skip quantization while converting to GGUF.",
-)
-@click.option(
-    "--model-name",
-    help="Name of the model being trained/converted. Informs the naming of the final trained model file",
-    default=None,
-    show_default=True,
+class SUPPORTED_FORMATS(Enum):
+    GGUF = "gguf"
+    SAFETENSORS = "safetensors"
```

Comment:
Does this include `.bin` as well?

Reply:
havent tested with .bin so im going to say not at the moment